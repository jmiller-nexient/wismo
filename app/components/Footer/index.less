@import '../../styles/themes/index.less';

.footer-container {
  background-color: @dte-background-brand-blue;
  bottom: 0px;
  height: 182px;
  width: 100%;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  flex-wrap: wrap;
}

.footer-links {
  display: flex;
  width: 25%;
  justify-content: space-evenly;
  height: 60px;
  flex-wrap: wrap;
  font-size: @dte-wismo-font-size-xs;
}

.footer-copyrights-notice {
  display: flex;
  flex-wrap: wrap;
  color: @dte-content-primary-inverse;
  font-size: @dte-wismo-font-size-xs;
}

a:link,
a:visited {
  color: @dte-content-primary-inverse  !important;
  text-decoration: none;
}

a:hover,
a:active {
  text-decoration: underline;
}
