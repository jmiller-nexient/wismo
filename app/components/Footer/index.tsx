import * as React from 'react';

import './index.less';

const Footer: React.FC = (): React.ReactElement => (
  <footer className="footer-container">
    <div className="footer-links">
      <a href="https://newlook.dteenergy.com/wps/wcm/connect/dte-web/quicklinks/footer/customer+data+privacy+policy">
        Privacy Policy
      </a>
      <a href="https://newlook.dteenergy.com/wps/wcm/connect/dte-web/quicklinks/footer/terms-and-conditions">
        Terms & Conditions
      </a>
      <a href="https://newlook.dteenergy.com/wps/wcm/connect/dte-web/quicklinks/footer/03+contact+us">Contact Us</a>
    </div>
    <div className="footer-copyrights-notice">
      <p>{new Date().getFullYear()} DTE Energy. All rights reserved</p>
    </div>
  </footer>
);

export default Footer;
