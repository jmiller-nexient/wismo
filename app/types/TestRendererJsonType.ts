import { ReactTestRendererJSON } from 'react-test-renderer';

type TestRendererJsonType = ReactTestRendererJSON | null;

export default TestRendererJsonType;
