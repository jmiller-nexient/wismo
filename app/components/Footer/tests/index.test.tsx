import React from 'react';
import renderer from 'react-test-renderer';

import TestRendererJsonType from 'types/TestRendererJsonType';

import Footer from '..';

describe('<Footer />', () => {
  it('should render correctly', () => {
    const tree: TestRendererJsonType = renderer.create(
      <Footer />,
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
