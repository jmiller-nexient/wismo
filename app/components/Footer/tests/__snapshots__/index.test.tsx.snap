// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render correctly 1`] = `
<footer
  className="dte-wismo-footer"
>
  <div
    className="dte-wismo-footer-links"
  >
    <a
      href="https://newlook.dteenergy.com/wps/wcm/connect/dte-web/quicklinks/footer/customer+data+privacy+policy"
    >
      Privacy Policy
    </a>
    <a
      href="https://newlook.dteenergy.com/wps/wcm/connect/dte-web/quicklinks/footer/terms-and-conditions"
    >
      Terms & Conditions
    </a>
    <a
      href="https://newlook.dteenergy.com/wps/wcm/connect/dte-web/quicklinks/footer/03+contact+us"
    >
      Contact Us
    </a>
  </div>
  <div
    className="dte-wismo-footer-copyright"
  >
    <p>
      2020
       DTE Energy. All rights reserved
    </p>
  </div>
</footer>
`;
